Wikipedia/Code/PreviewLicenseView.m

```diff
@@ -81,7 +81,8 @@ - (void)styleLinks:(UILabel *)label {
     label.attributedText =
         [label.text attributedStringWithAttributes:baseAttributes
                                substitutionStrings:@[WMFLocalizedStringWithDefaultValue(@"wikitext-upload-save-terms-name", nil, nil, @"Terms of Use", @"This message is used in the message [[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-and-license]].\n{{Identical|Terms of use}}"),
-                                                     WMFLocalizedStringWithDefaultValue(@"wikitext-upload-save-license-name", nil, nil, @"CC BY-SA 3.0", @"Name of license user edits are saved under - presently CC BY-SA 3.0\n{{Identical|CC BY-SA}}")]
+                                                     WMFLocalizedStringWithDefaultValue(@"wikitext-upload-save-license-name", nil, nil, @"CC BY-SA 3.0", @"Name of license user edits are saved under - presently CC BY-SA 3.0\n{{Identical|CC BY-SA}}"),
+                                                     WMFLocalizedStringWithDefaultValue(@"wikitext-upload-save-license-name2", nil, nil, @"GFDL", @"Name of the second license user edits are saved under - presently GFDL\n{{Identical|GFDL}}")]]
```

Comment:
The code needs to be built through Xcode after strings are added so that they get added to the appropriate strings files, this one appears to be missing. Also, could you change the key to `gdfl` instead of `wikitext-upload-save-license-name2`?